packages/atomic-react/src/components/stencil-generated/index.ts

```diff
@@ -0,0 +1,69 @@
+export const AtomicResultSectionEmphasized = /*@__PURE__*/createReactComponent<JSX.AtomicResultSectionEmphasized, HTMLAtomicResultSectionEmphasizedElement>('atomic-result-section-emphasized');
+export const AtomicResultSectionExcerpt = /*@__PURE__*/createReactComponent<JSX.AtomicResultSectionExcerpt, HTMLAtomicResultSectionExcerptElement>('atomic-result-section-excerpt');
+export const AtomicResultSectionTitle = /*@__PURE__*/createReactComponent<JSX.AtomicResultSectionTitle, HTMLAtomicResultSectionTitleElement>('atomic-result-section-title');
+export const AtomicResultSectionTitleMetadata = /*@__PURE__*/createReactComponent<JSX.AtomicResultSectionTitleMetadata, HTMLAtomicResultSectionTitleMetadataElement>('atomic-result-section-title-metadata');
+export const AtomicResultSectionVisual = /*@__PURE__*/createReactComponent<JSX.AtomicResultSectionVisual, HTMLAtomicResultSectionVisualElement>('atomic-result-section-visual');
+export const AtomicResultTemplate = /*@__PURE__*/createReactComponent<JSX.AtomicResultTemplate, HTMLAtomicResultTemplateElement>('atomic-result-template');
+export const AtomicResultText = /*@__PURE__*/createReactComponent<JSX.AtomicResultText, HTMLAtomicResultTextElement>('atomic-result-text');
+export const AtomicResultsPerPage = /*@__PURE__*/createReactComponent<JSX.AtomicResultsPerPage, HTMLAtomicResultsPerPageElement>('atomic-results-per-page');
+export const AtomicSearchBox = /*@__PURE__*/createReactComponent<JSX.AtomicSearchBox, HTMLAtomicSearchBoxElement>('atomic-search-box');
+export const AtomicSearchBoxQuerySuggestions = /*@__PURE__*/createReactComponent<JSX.AtomicSearchBoxQuerySuggestions, HTMLAtomicSearchBoxQuerySuggestionsElement>('atomic-search-box-query-suggestions');
+export const AtomicSearchBoxRecentQueries = /*@__PURE__*/createReactComponent<JSX.AtomicSearchBoxRecentQueries, HTMLAtomicSearchBoxRecentQueriesElement>('atomic-search-box-recent-queries');
+export const AtomicSearchInterface = /*@__PURE__*/createReactComponent<JSX.AtomicSearchInterface, HTMLAtomicSearchInterfaceElement>('atomic-search-interface');
+export const AtomicSortDropdown = /*@__PURE__*/createReactComponent<JSX.AtomicSortDropdown, HTMLAtomicSortDropdownElement>('atomic-sort-dropdown');
+export const AtomicSortExpression = /*@__PURE__*/createReactComponent<JSX.AtomicSortExpression, HTMLAtomicSortExpressionElement>('atomic-sort-expression');
+export const AtomicTableElement = /*@__PURE__*/createReactComponent<JSX.AtomicTableElement, HTMLAtomicTableElementElement>('atomic-table-element');
+export const AtomicText = /*@__PURE__*/createReactComponent<JSX.AtomicText, HTMLAtomicTextElement>('atomic-text');
+export const AtomicTimeframe = /*@__PURE__*/createReactComponent<JSX.AtomicTimeframe, HTMLAtomicTimeframeElement>('atomic-timeframe');
+export const AtomicTimeframeFacet = /*@__PURE__*/createReactComponent<JSX.AtomicTimeframeFacet, HTMLAtomicTimeframeFacetElement>('atomic-timeframe-facet');
```

Comment:
This is for another PR, but as atomic changes, what would the process be to regenerate files inside `stencil-generated`? e.g. CI/CD job that runs on PRs and pushes to the open branch?

Reply:
I'll think about it, and tackle in another PR.